### ISSUE TITLE
Questions about distribution alignment

### ISSUE THREAD

**Issue by omg777** [zysymu/AdaMatch-pytorch#1]
Hi, thanks for sharing you codes.
I have a question about distribution alignment.
What's the meaning of `softmax(dim=0)` ? I don't understand `softmax with dim=0` not just `dim=1`.
```
...
adamatch.py
...
pseudolabels_source = F.softmax(logits_source_weak, 0) <--- Here

  ## softmax for logits of weakly augmented target images
  logits_target = logits_combined[source_total:]
  logits_target_weak = logits_target[:data_target_weak.size(0)]
  pseudolabels_target = F.softmax(logits_target_weak, 0) <--- Here
```

**Comment by zysymu** [comment]
hi, sorry for the huge delay, I didn't see the issue notification!
and indeed it should be `dim=1` in both lines. I fixed this and some other problems that my code had and I'm now uploading the new version
